clients/go/README.md

```diff
@@ -12,13 +12,7 @@ In order to do this please follow [this guide](../../gateway-protocol-impl/READM
 To regenerate the gateway mock `internal/mock_pb/mock_gateway.go` run [`mockgen`](https://github.com/golang/mock#installation):
 
 ```
-GO111MODULE=off mockgen github.com/camunda/zeebe/clients/go/v8/pkg/pb GatewayClient,Gateway_ActivateJobsClient > internal/mock_pb/mock_gateway.go
-```
-
-If you see errors regarding packages which are not found then as alternative you can try this:
-
-```
-GO111MODULE=off mockgen -source $GOPATH/src/github.com/camunda/zeebe/clients/go/v8/pkg/pb/gateway.pb.go GatewayClient,Gateway_ActivateJobsClient > internal/mock_pb/mock_gateway.go
+mockgen -source=pkg/pb/gateway_grpc.pb.go GatewayClient,Gateway_ActivateJobsClient > internal/mock_pb/mock_gateway.go
```

Comment:
🔧 Before it was using the GOPATH, which was kind of an absolut path, now we using an relative path. Would be good if we then mention that you have to be inside `clients/go` to run this.

Maybe the same what you wrote here https://camunda.slack.com/archives/C05DH1F5TAR/p1702311839690769?thread_ts=1702306903.159889&cid=C05DH1F5TAR